src/homepageExperience/components/steps/cli/ExecuteAggregateQuery.tsx

```diff
@@ -71,7 +72,11 @@ export const ExecuteAggregateQuery = (props: OwnProps) => {
       />
       <p>In the InfluxDB CLI, run the following:</p>
       <CodeSnippet
-        text={codeSnippet}
+        text={
+          window?.navigator?.userAgent.includes('Windows')
```

Comment:
should we put this into a utility function? i remember you made a utility to detect ctrl-c keyboard, seems like a good candidate to put in there.